lib/Controller/QueueController.php

```diff
@@ -93,19 +93,21 @@ public function __construct(
 	 * @param string $token
 	 */
 	public function asyncWithToken(string $token) {
-		$this->async();
-
 		$requests = $this->queueService->getRequestFromToken($token, RequestQueue::STATUS_STANDBY);
-		$this->activityService->manageInit();
-		foreach ($requests as $request) {
-			$request->setTimeout(ActivityService::TIMEOUT_ASYNC);
-			try {
-				$this->activityService->manageRequest($request);
-			} catch (RequestException $e) {
-			} catch (SocialAppConfigException $e) {
+
+		if (sizeof($requests) > 0) {
```

Comment:
`if (!empty($requests)) {`

You dont need to count the number, you just want to know if its empty or not